### PR TITLE
docs(zh,en): Update translation of core-concepts/index, move SSR link forward to new mention

### PR DESCRIPTION
**Pull request by PrainLopez**
`packages/docs/zh/core-concepts/index.md`
更新翻译同步英文文档现版本，个人经验不足，72行需要一些review
Update the docs/zh translation to latest. May need review at line 72 due to my experience.

`packages/docs/core-concepts/index.md`
In the process of translating I found a bug, the line 71 was added after line 73 while the SSR link is remained. Has moved it forward and cancelled the latter one on line 73.

### CODE REVIEW

File: packages/docs/zh/core-concepts/index.md [hunk 1]
```diff
@@ -64,7 +65,32 @@ export const useCounterStore = defineStore('counter', () => {
 - `computed()` 就是 `getters`
 - `function()` 就是 `actions`
 
-Setup store 比 [Option Store](#option-stores) 带来了更多的灵活性，因为你可以在一个 store 内创建侦听器，并自由地使用任何[组合式函数](https://cn.vuejs.org/guide/reusability/composables.html#composables)。不过，请记住，使用组合式函数会让 [SSR](../cookbook/composables.md) 变得更加复杂。
+注意，要让 pinia 正确识别 `state`，你**必须**在setup store中返回 **`state` 的所有属性**。这意味着，你不能在 store 中使用**私有**属性。不完整返回会影响 [SSR](../cookbook/composables.md) ，开发工具和其他插件的正常运行。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/core-concepts/index.md]:
```suggestion
注意，要让 pinia 正确识别 `state`，你**必须**在 setup store 中返回 **`state` 的所有属性**。这意味着，你不能在 store 中使用**私有**属性。不完整返回会影响 [SSR](../cookbook/composables.md) ，开发工具和其他插件的正常运行。
```

File: packages/docs/zh/core-concepts/index.md [hunk 2]
```diff
@@ -64,7 +65,32 @@ export const useCounterStore = defineStore('counter', () => {
...
+  // ...
+
+  return {
+    // ...
+  }
+})
+```
+
+:::warning
+不要返回像 `route` 或 `appProvided` (上例中)之类的属性，因为它们不属于 store ，而且你可以在组建中直接用 `useRoute()` 和 `inject('appProvided')` 访问。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/core-concepts/index.md]:
```suggestion
不要返回像 `route` 或 `appProvided` (上例中)之类的属性，因为它们不属于 store，而且你可以在组件中直接用 `useRoute()` 和 `inject('appProvided')` 访问。
```

File: packages/docs/zh/core-concepts/index.md [hunk 3]
```diff
@@ -64,7 +65,32 @@ export const useCounterStore = defineStore('counter', () => {
 - `computed()` 就是 `getters`
 - `function()` 就是 `actions`
 
-Setup store 比 [Option Store](#option-stores) 带来了更多的灵活性，因为你可以在一个 store 内创建侦听器，并自由地使用任何[组合式函数](https://cn.vuejs.org/guide/reusability/composables.html#composables)。不过，请记住，使用组合式函数会让 [SSR](../cookbook/composables.md) 变得更加复杂。
+注意，要让 pinia 正确识别 `state`，你**必须**在 setup store 中返回 **`state` 的所有属性**。这意味着，你不能在 store 中使用**私有**属性。不完整返回会影响 [SSR](../cookbook/composables.md) ，开发工具和其他插件的正常运行。
+
+Setup store 比 [Option Store](#option-stores) 带来了更多的灵活性，因为你可以在一个 store 内创建侦听器，并自由地使用任何[组合式函数](https://cn.vuejs.org/guide/reusability/composables.html#composables)。不过，请记住，使用组合式函数会让 SSR 变得更加复杂。
+
+Setup stores 也可以依赖于全局**提供**的属性，比如路由。任何[应用层面提供](https://vuejs.org/api/application.html#app-provide)的属性都可以在 store 中使用 `inject()` 访问，就像在组件中一样：
```
Comment (awxiaoxian2020) [inline packages/docs/zh/core-concepts/index.md]:
```suggestion
Setup store 也可以依赖于全局**提供**的属性，比如路由。任何[应用层面提供](https://vuejs.org/api/application.html#app-provide)的属性都可以在 store 中使用 `inject()` 访问，就像在组件中一样：
```

File: packages/docs/zh/core-concepts/index.md [hunk 4]
```diff
@@ -82,9 +108,11 @@ const store = useCounterStore()
 </script>
 ```
 
-你可以定义任意多的 store，但为了让使用 pinia 的益处最大化(比如允许构建工具自动进行代码分割以及 TypeScript 推断)，**你应该在不同的文件中去定义 store**。
-
+:::tip
 如果你还不会使用 `setup` 组件，[你也可以通过**映射辅助函数**来使用 Pinia](../cookbook/options-api.md)。
+:::
+
+你可以定义任意多的 store，但为了让使用 pinia 的益处最大化(比如允许构建工具自动进行代码分割以及 TypeScript 推断)，**你应该在不同的文件中去定义 store**。
```
Comment (awxiaoxian2020) [inline packages/docs/zh/core-concepts/index.md]:
```suggestion
你可以定义任意多的 store，但为了让使用 pinia 的益处最大化 (比如允许构建工具自动进行代码分割以及 TypeScript 推断)，**你应该在不同的文件中去定义 store**。
```

**Review by awxiaoxian2020 — COMMENTED** [review]
LGTM!

**Comment by PrainLopez** [comment]
Screenshot 2024-01-19 092350

> `packages/docs/core-concepts/index.md`
>In the process of translating I found a bug, the line 71 was added after line 73 while the SSR link is remained. Has moved it >forward and cancelled the latter one on line 73.

This is now causing conflict, should I submit it in a seperate PR as it's issue of EN docs?
(add), I've already done this link move in `docs/zh`, I know that by any result keeping docs sync is priority

**Review by awxiaoxian2020 — COMMENTED** [review]
LGTM

**Pull request merged**